dear_petition/petition/migrations/0021_merge_20200425_1737.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 2.2.10 on 2020-04-25 17:37
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("petition", "0020_comment_time"),
+        ("petition", "0020_offense_offenserecord"),
```

Comment:
I think we should cherry-pick this `commit cf993dcc69318d09082458bd8bdb0e3bab8d09a2` from the upgrade branch. It re-orders the comment migrations putting them after the offense/offenserecord migration (21-23)